packages/typespec-vscode/test/extension/create-typespec.test.ts

```diff
@@ -57,10 +60,141 @@ const CreateCasesConfigList: CreateConfigType[] = [
   },
 ];
 
+templateName = "Typespec library";
```

Comment:
can we reduce a lot the number of test(there should be at most one per action). this is a lot of wasted time. UI test should just be a small check that everything is still working. make unit test if you need more grannular things.

Reply:
@timotheeguerin Do you have any thoughts on the 9 UI cases to be covered above?

Reply:
@timotheeguerin According to your suggestion, the current PR only covers 4 cases.